README.md

```diff
@@ -66,7 +66,13 @@ a number of programs to be installed:
 
 -   [Template Toolkit].  The Debian package is `libtemplate-perl`
 -   [Pandoc].  The Debian package is `pandoc`
--   git
+-   `make`, `wget`, `git`, `perl`, `python3`, `gcc`, `libssl-dev`, and `libz-dev`
+
+The building procedure also requires the following Perl modules to be available:
+-   `Module::Install`,
+-   `URI::Encode`, `Net::SSLeay`, `IO::Socket::SSL`,
+-   `LWP::UserAgent`, `LWP::Protocol`
```

Comment:
Are we using any of those here?  It's true that at least some of them are needed to build OpenSSL::Query, but that belongs *there*, not here.  Here, it should suffice to depend on OpenSSL::Query.

Reply:
That's actually true. Not sure if every module was required to get ::Query one, but I feel like it is.